modules/adding-a-custom-model-serving-runtime.adoc

```diff
@@ -25,7 +25,11 @@ NOTE: {productname-short} includes the OpenVINO Model Server model-serving runti
 . From the {productname-short} dashboard, click *Settings* > *Serving runtimes*.
 +
 The *Serving runtimes* page opens and shows the model-serving runtimes that are already installed and enabled in your {productname-short} deployment. By default, the OpenVINO Model Server runtime is pre-installed and enabled in {productname-short}.
-. To add a new, custom runtime, click *Add serving runtime*.
+. To add a custom runtime:
```

Comment:
Suggest introducing list with complete sentence. "To add a custom runtime, choose one of the following options:"